File: InMotionGIT.Common - Copy/Services/Interfaces/IRestDataManager.vb
Imports System.ComponentModel
Imports System.IO
Imports System.ServiceModel
Imports System.ServiceModel.Web

Namespace Services.Interfaces

    <ServiceContract()>
    Public Interface IRestDataManager

        <WebInvoke(Method:="GET", RequestFormat:=WebMessageFormat.Json, ResponseFormat:=WebMessageFormat.Json, BodyStyle:=WebMessageBodyStyle.WrappedRequest)>
        <OperationContract()>
        <Description("Ramos activos.")>
        Function Query(command As String) As Stream

    End Interface

End Namespace

File: Common/InMotionGIT.Common/Services/Interfaces/IRestDataManager.cs
using System.ComponentModel;
using System.IO;
using System.ServiceModel;
using System.ServiceModel.Web;

namespace InMotionGIT.Common.Services.Interfaces
{

    [ServiceContract()]
    public interface IRestDataManager
    {

        [WebInvoke(Method = "GET", RequestFormat = WebMessageFormat.Json, ResponseFormat = WebMessageFormat.Json, BodyStyle = WebMessageBodyStyle.WrappedRequest)]
        [OperationContract()]
        [Description("Ramos activos.")]
        Stream Query(string command);

    }

}